eftest/src/eftest/runner.clj

```diff
@@ -5,10 +5,24 @@
             [clojure.tools.namespace.find :as find]
             [eftest.report :as report]
             [eftest.report.progress :as progress]
-            [eftest.output-capture :as capture]))
+            [eftest.output-capture :as capture])
+  (:import [java.util.concurrent Executors ExecutorService]))
 
 (defmethod test/report :begin-test-run [_])
 
+(defn make-threadpool
+  ([] (make-threadpool (.availableProcessors (Runtime/getRuntime))))
```

Comment:
Maybe make it `(+ 2 (.availableProcessors (Runtime/getRuntime)))` for consistency with the inbuilt Clojure threadpool.